### ISSUE TITLE
[Enhancement] Hwyla/Wthit compatibility for magma flower, feather fern, ink bush and black apple

### ISSUE THREAD

**Issue by saltyseadoggo** [paulevsGitch/BetterNether#322]
The magma flower, feather fern, ink bush and black apple plants are like crops in that the player must wait for them to reach their final `age` value before harvesting them to receive the maximum produce from them. With vanilla crops, HWYLA and, more recently with its abandoning, WTHIT show their maturity percentage so that you know when it's time to harvest. I would like to see this feature be made available to the four plants listed above so that I can tell when they need to be harvested, as, right now, this requires memorizing what they look like in their final stage, or what their `age` blockstate in the F3 menu should be.

Basically, show the 'growth' line from vanilla crops...
![2021-03-04_13 10 52](https://user-images.githubusercontent.com/69029346/110030939-2a135180-7ceb-11eb-8e17-04ebef42e0a1.png)
... on these.
![2021-03-04_13 11 02](https://user-images.githubusercontent.com/69029346/110030958-2ed80580-7ceb-11eb-8c37-63316a8f07e0.png)

**Comment by quiqueck** [comment]
Moving to our Enhancement List.
